theme/theme.go

```diff
@@ -169,6 +169,8 @@ func (t *builtinTheme) Size(s fyne.ThemeSizeName) float32 {
 		return 5
 	case SizeNameSelectionRadius:
 		return 3
+	case SizeNameScrollBarRadius:
+		return 4
```

Comment:
is 4 right? Should it not match the selection radius or size of the slider bars?
I have not looked at it rendered, but 3 seems right mathematically...

Reply:
4 was an arbitrary choice. I can switch it to 3